TESTING.md

```diff
@@ -3,44 +3,59 @@
 Currently, there are no automated tests. The plan would be to automate the manual steps via travis.
 
 ## Manual
-The below steps can be used to test each pipeline lib method.
+The below steps can be used to test each pipeline lib method. If you are testing a PR or your local branch, you need to change the following:
+- test/create-pipeline-library.groovy - to point to your git repo, instead of https://github.com/redhat-cop/pipeline-library.git
+- test/Jenkinsfile-* - the branch for the Jenkinsfile needs to match the branch of the changes
 
 ### Create projects
+```bash
 oc new-project pipelinelib-testing
+```
 
 ### Import additional image streams for slaves
 // Use v1.11 until: https://github.com/redhat-cop/containers-quickstarts/pull/243
```

Comment:
Since you're updating the instructions, would you mind removing this comment and change to either `v1.15` or `latest` below?

Reply:
pushed